interactions/api/error.py

```diff
@@ -377,4 +260,25 @@ def lookup() -> dict:
             170007: "Sticker animation duration exceeds maximum of 5 seconds",
             180000: "Cannot update a finished event",
             180002: "Failed to create stage needed for stage event",
-        }
+        }.get(code, f"Unknown error: {code}")
+
+    def __init__(self, message: str = None, code: int = 0, severity: int = 0, **kwargs):
+        self.code: int = code
+        self.severity: int = severity
+        self.data: dict = kwargs.pop("data", None)
+        self.message: str = message or self.lookup(self.code)
+        _fmt_error: List[tuple] = []
+
+        if (
+            self.data
+            and isinstance(self.data, dict)
+            and isinstance(self.data.get("errors", None), dict)
+        ):
+            _fmt_error: List[tuple] = [i for i in self._parse(self.data["errors"])]
```

Comment:
Can't you just do this:
```suggestion
            _fmt_error: List[tuple] = list(self._parse(self.data["errors"]))
```

Reply:
My own code is making my brain hurt. Lemme see

Reply:
wait a sec, _parse returns a list

Reply:
Do this instead
```suggestion
            _fmt_error: List[tuple] = self._parse(self.data["errors"])
```

Reply:
Yep

Reply:
So this is probably just redundant